types/swagger2openapi/index.d.ts

```diff
@@ -0,0 +1,145 @@
+// Type definitions for swagger2openapi 7.0
+// Project: https://github.com/Mermade/oas-kit/tree/main/packages/swagger2openapi
+// Definitions by: rxliuli <https://github.com/rxliuli>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Definitions by: rxliuli <https://github.com/rxliuli>
+/// <reference types="node" />
```

Comment:
```suggestion

/// <reference types="node" />
```
(just for readability)